assets/js/Sign.jsx

```diff
@@ -144,6 +144,13 @@ class Sign extends Component {
             <span style={fontSize(signId)}>
               {signId}
             </span>
+            {readOnly
+              && (
+                <div className="viewer--mode-text">
+                  {signConfig.mode}
```

Comment:
Minor usability improvement idea: map this to the same value that would actually appear in the dropdown (i.e. "Custom" instead of "static_text").